db/db.go

```diff
@@ -6,37 +6,36 @@ import (
 	"os"
 	"strings"
 
-	_ "github.com/mattn/go-sqlite3"
-
-	log "code.google.com/p/log4go"
-)
-
-const (
-	dbName = "db.sqlite"
+	_ "github.com/mattn/go-sqlite3" // required blank import
+	"github.com/otoolep/rqlite/log"
 )
 
-// The SQL database.
+// DB is the SQL database.
 type DB struct {
 	dbConn *sql.DB
 }
 
-// Query result types
+// RowResult is the result type.
 type RowResult map[string]string
+
+// RowResults is the list of results.
 type RowResults []map[string]string
 
 // New creates a new database. Deletes any existing database.
 func New(dbPath string) *DB {
-	log.Trace("Removing any existing SQLite database at %s", dbPath)
-	os.Remove(dbPath)
+	log.Tracef("Removing any existing SQLite database at %s", dbPath)
+	if err := os.Remove(dbPath); err != nil {
```

Comment:
kk, I can ignore this error using a `_ = os.Remove(...` notation, so the code reader knows that this function returns an error value, but it's unimportant, and make the linter happy.